api/app/signals/apps/dsl/grammar.tx

```diff
@@ -0,0 +1,12 @@
+RootExpression: expression = OrExpression;
```

Comment:
Is this file actually used? The `ExpressionEvaluator` class uses a locally defined string `GRAMMAR` in `api/app/signals/apps/dsl/ExpressionEvaluator.py` to create a *textX* meta model. Remove?

Reply:
deleted